Makefile

```diff
@@ -24,8 +27,13 @@ VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 # go build common flags
 BUILDFLAGS := -buildmode=pie
 
+# whether stipping the binary
+ifeq ($(STRIP),yes)
+       LDFLAGS += -w -s
```

Comment:
Could you rename to KATA_LDFLAGS ? we had some build problems at pre-release steps https://github.com/kata-containers/runtime/issues/2478

Reply:
sure